sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -602,6 +607,15 @@ export class ApplicationOfferingChangeRequestService {
           submittedBy: auditUser,
           submittedDate: currentDate,
         } as StudentAssessment;
+        // Update the student appeal record for the student assessment if it exists.
+        if (studentAssessment?.studentAppeal) {
```

Comment:
No need to copy the `application.currentAssessment`, the property can be directly accessed as below.
```ts
// Update the student appeal record for the student assessment if it exists.
const studentAssessment = application.currentAssessment;
if (studentAssessment.studentAppeal) {
  application.currentAssessment.studentAppeal = {
    id: studentAssessment.studentAppeal.id,
  } as StudentAppeal;
}
```

Reply:
I am not able to see this change. Is it pending an update?

Reply:
It has been updated. This [commit](https://github.com/bcgov/SIMS/pull/2961/commits/c1644afcb55de6524df27c1c093cd7802925e009) has the update.

Reply:
I still can see it in the shared commit and GitHub file, as shown below.
![image](https://github.com/bcgov/SIMS/assets/61259237/c321bebd-6d2e-49e4-b593-22588487837a)